src/Symfony/Component/Form/FormRegistry.php

```diff
@@ -23,35 +23,28 @@
  */
 class FormRegistry implements FormRegistryInterface
 {
-    /**
-     * @var FormExtensionInterface[]
-     */
-    private array $extensions = [];
-
     /**
      * @var ResolvedFormTypeInterface[]
      */
     private array $types = [];
 
     private FormTypeGuesserInterface|null|false $guesser = false;
-    private ResolvedFormTypeFactoryInterface $resolvedTypeFactory;
     private array $checkedTypes = [];
 
     /**
      * @param FormExtensionInterface[] $extensions
      *
      * @throws UnexpectedTypeException if any extension does not implement FormExtensionInterface
      */
-    public function __construct(array $extensions, ResolvedFormTypeFactoryInterface $resolvedTypeFactory)
-    {
+    public function __construct(
+        private array $extensions,
```

Comment:
I would revert this change. In other places where we check the passed value to (for example) throw exceptions we don't use CPP.

Reply:
If I understood @nicolas-grekas correct, he explicitly wants that, but I am fine either way

Reply:
I'm fine either way, but we should then be consistent IMO

Reply:
Can we use CPP is the classes @xabbuh mentions?
Also, please don't make pinging me an argument from authority, it's not and I don't want it to be ever. Better think and explain, and convince.